### ISSUE TITLE
The name property should be different in the same page

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1949]
URL: http://clarity-ks.surge.sh/progress-bars

Tools: Web Accessibility Toolbar (WAT2.0) / Inspect32

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Progress tab.
3. Check the name property of four SHOW buttons in Progress tab.

Actual Results:
The name property of four SHOW buttons is “Show”.  
It’s difficult to distinguish between them for Blind.
For more details, please refer to “Progress-SHOW-NamePropertyIsSame.png”.

Expected Results:
The name property should be different in the same page.
The name property of four SHOW buttons in Progress tab should be different and identify their own purpose.

Note:
This issue occurs in other pages, such as Alerts, Checkbox, Radios, Vertical Nav, Wizards.

Fix Suggestion:
1. Set the name property of SHOW for “Labeled w/ Fade Out” to “Show Labeled w/ Fade Out”.
2. Set the name property of SHOW for “Success Flash w/ fade” to “Show Success Flash w/ fade”.
3. Set the name property of SHOW for “Danger/Warning” to “Show Danger/Warning”.
4. Set the name property of SHOW for “Fade Out, flash danger” to “Show Fade Out, flash danger”.

Violation of Section 508 Standards: 
1194.21(d) Sufficient information about a user interface element including the identity, operation and state of the element shall be available to assistive technology. When an image represents a program element, the information conveyed by the image must also be available in text. 

User Impact: Blind.

Screen shot:

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/e7cc6109-ed63-4e61-b710-f9d5d7b84ee3)

NOTE: this is an implementation issue.
